From b2058cea1811373e77473bdc89147d8d4a04779c Mon Sep 17 00:00:00 2001
From: Panu Matilainen <pmatilai@redhat.com>
Date: Tue, 30 Apr 2024 12:32:16 +0300
Subject: [PATCH] Don't emit target build into on --quiet

Kinda related to 1261b413d0380289623a8ac695279b0de82018c6,
quiet means quiet and since this isn't using rpmlog() we need to filter
manually.
---
 tools/rpmbuild.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/tools/rpmbuild.c b/tools/rpmbuild.c
index 0a57395dbe..bb47e99613 100644
--- a/tools/rpmbuild.c
+++ b/tools/rpmbuild.c
@@ -549,8 +549,8 @@ static int build(rpmts ts, const char * arg, BTA_t ba, const char * rcfile)
     }
 
     /* parse up the build operators */
-
-    fprintf(stderr, _("Building target platforms: %s\n"), targets);
+    if (!quiet)
+	fprintf(stderr, _("Building target platforms: %s\n"), targets);
 
     ba->buildAmount &= ~buildCleanMask;
     for (ARGV_const_t target = build_targets; target && *target; target++) {
@@ -558,7 +558,8 @@ static int build(rpmts ts, const char * arg, BTA_t ba, const char * rcfile)
 	if (*(target + 1) == NULL)
 	    ba->buildAmount |= cleanFlags;
 
-	fprintf(stderr, _("Building for target %s\n"), *target);
+	if (!quiet)
+	    fprintf(stderr, _("Building for target %s\n"), *target);
 
 	/* Read in configuration for target. */
 	rpmFreeMacros(NULL);
